vignettes/4_shallowhrd-vignette.Rmd

```diff
@@ -0,0 +1,249 @@
+
+---
+
+### II - Input data
+
+---
+
+As an example dataset we will again use the samples found in the *utanosmodellingdata* repository found [here](https://github.com/Huntsmanlab/utanosmodellingdata). If not already done, clone that repo to somewhere convenient such as a common 'repos' folder on your machine and read in the example data. It is human endometrial carcinoma sWGS data aligned to hg19. The associated publication [@Jamieson2024] was published in CCR and can be found [here](https://doi.org/10.1158/1078-0432.CCR-23-3689).
+
+Download said data/clone the repo, and read in the relative copy-numbers:  
+```{r setup, eval=FALSE}
+> library(utanos)
+> library(QDNAseq)
+> library(ggplot2)
+> rcn.obj <- readRDS("~/repos/utanosmodellingdata/sample_copynumber_data/sample_rcn_data.rds")
+```
+
+```{r, eval = FALSE}
+> rcn.obj
+QDNAseqCopyNumbers (storageMode: lockedEnvironment)
+assayData: 103199 features, 10 samples 
+  element names: copynumber, segmented 
+protocolData: none
+phenoData
+  sampleNames: CC-CHM-1341 CC-CHM-1347 ... CC-HAM-0385 (10 total)
+  varLabels: name total.reads ... loess.family (6 total)
+  varMetadata: labelDescription
+featureData
+  featureNames: 1:1-30000 1:30001-60000 ... Y:59370001-59373566 (103199 total)
+  fvarLabels: chromosome start ... comCNV.mask (11 total)
+  fvarMetadata: labelDescription
+experimentData: use 'experimentData(object)'
+Annotation: 
+```
+
+---
+
+### III - Extract data and run shallowHRD
+
+---
+
+We can run shallowHRD directly on the QDNAseq object we just read in, using the `RunShallowHRDFromQDNA()` function, or on a table of segments, using the `RunShallowHRD()` function. Since estimating the threshold used to merge segments requires us to take 100,000 samples by default, running a single sample can take a few minutes. In the interest of time, we will extract a single sample from the QDNAseq object and demonstrate running shallowHRD on a table of segments. To run it for all the samples in the object, we could just call `RunShallowHRDFromQDNA()` directly on the QDNAseq object, with most arguments remaining the same as those we'll provide to `RunShallowHRD()`.
```

Comment:
The second sentence of this paragraph is a bit confusing. 100,000 samples of what? Are these 100,000 samples the the same as the 'sample' written later in the sentence?

Reply:
Reworded this.